pseudocode.md

```diff
@@ -3,4 +3,45 @@
 ```
 Write pseudocode here, within the tripple backticks.
 
+ask user for a word
+for each letter in the word:
+  print out a dash
+  
+print out gallows
+print out empty word bank 
+win is false
+
+while man is not fully hanged:
+  ask user to guess a letter
+  
+  if the input is longer than one character and the input is not the word:
```

Comment:
There could be a debate whether or not one loses if they incorrectly guess the whole word.  Think of a way with our current model, how we would keep track of word-guesses and have that add one more body part.